.gitignore

```diff
@@ -7,13 +7,19 @@ cpp/unreal_plugins/*/ThirdParty
 
 cpp/unreal_projects/SpearSim/Binaries
 cpp/unreal_projects/SpearSim/Build
+cpp/unreal_projects/SpearSim/Content/__ExternalActors__
```

Comment:
I understand that `Engine/Content/__ExternalActors__` and `Engine/Content/__ExternalObjects__` are not auto-generated. They are present in `Engine/Content` immediately after installing the engine. And I think they're used for the third-person map because this map has the _one asset per file_ option switched on. But are we sure that the copied third-person map somehow knows to refer to the copied `__ExternalActors__` and `__ExternalObjects__` folders? Or does it still refer to the folders in `Engine/Content`.

Reply:
Resolved. Current behavior in PR is correct.